src/components/booking/ChooseTickets.vue

```diff
@@ -0,0 +1,249 @@
+<script setup lang="ts">
+import { Ref, ref } from "vue";
+import { storeToRefs } from "pinia";
+import { useRouter } from "vue-router";
+import { useReservationStore } from "@/stores/reservation.js";
+import { useAuthStore } from "@/stores/auth.js";
+import BaseButton from "@/components/global/BaseButton.vue";
+import BaseCheckbox from "@/components/global/BaseCheckbox.vue";
+
+export interface ReservedSeatTicket {
+  seat: string;
+  ticket_type_id: number;
+}
+
+// hardcoded data cuz there is no tickets endpoint in the API
+const ticketTypes = [
+  { id: 1, type: "Adult", price: 13, label: "Adult - $13" },
+  { id: 2, type: "Student", price: 9, label: "Student - $9" },
+  { id: 3, type: "Senior", price: 7, label: "Senior - $7" },
+  { id: 4, type: "Child", price: 5, label: "Child - $5" },
+];
+
+const reservationStore = useReservationStore();
+const { reservedSeats } = storeToRefs(reservationStore);
+const authStore = useAuthStore();
+const { isUserLoggedIn } = storeToRefs(authStore);
+
+const router = useRouter();
+
+const reservedSeatsTickets: Ref<ReservedSeatTicket[]> = ref(
+  reservedSeats.value.map((reservedSeat) => {
+    return { seat: reservedSeat, ticket_type_id: 1 };
+  })
+);
+
+const terms: Ref<boolean> = ref(false);
```

Comment:
No need to be *that* explicit - if we pass a boolean value, TypeScript will know everything :)
```suggestion
const terms = ref(false);
```